public/index.html

```diff
@@ -3,10 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#325">
-
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
```

Comment:
No more favicon?

Reply:
Actually, `html-webpack-plugin` was loading it from `public` but now I think it's better to add the favicon in index.html itself. Adding it back.